src/app/signIn/signIn.component.js

```diff
@@ -0,0 +1,32 @@
+import angular from 'angular';
+import 'angular-gettext';
+
+import signInForm from 'common/components/signInForm/signInForm.component';
+
+import template from './signIn.tpl';
+
+let componentName = 'signIn';
+
+class SignInController {
+
+  /* @ngInject */
+  constructor( $log, $location ) {
+    this.$log = $log;
```

Comment:
Currently unused but logging errors here would be nice. It can stay if that's the intention eventually.